protocol/x/revshare/client/cli/query_params.go

```diff
@@ -0,0 +1,68 @@
+package cli
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/dydxprotocol/v4-chain/protocol/x/revshare/types"
+	"github.com/spf13/cobra"
+)
+
+func CmdQueryRevShareParams() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "revshare-params",
+		Short: "revshare params",
+		RunE: func(cmd *cobra.Command, args []string) (err error) {
+			clientCtx := client.GetClientContextFromCmd(cmd)
+			queryClient := types.NewQueryClient(clientCtx)
+			res, err := queryClient.MarketMapperRevenueShareParams(
+				context.Background(),
+				&types.QueryMarketMapperRevenueShareParams{},
+			)
+			if err != nil {
+				return err
+			}
+			return clientCtx.PrintProto(res)
+		},
+	}
+
+	flags.AddQueryFlagsToCmd(cmd)
+
+	return cmd
+}
+
+func CmdQueryRevShareDetailsForMarket() *cobra.Command {
```

Comment:
for my understanding, what does this return

Reply:
updated description with this